runtime/src/accounts_db.rs

```diff
@@ -9948,17 +9994,35 @@ pub mod tests {
         let entry2 = Arc::new(AccountStorageEntry::new(
             dummy_path, dummy_slot, dummy_id2, dummy_size,
         ));
-        entry2.alive_bytes.store(3000, Ordering::Relaxed);
+        entry2.alive_bytes.store(0, Ordering::Relaxed);
         candidates
             .entry(dummy_slot)
             .or_default()
             .insert(entry2.append_vec_id(), entry2.clone());
 
-        let output_candidates =
-            accounts.select_candidates_by_total_usage(&candidates, DEFAULT_ACCOUNTS_SHRINK_RATIO);
+        let dummy_id3 = 55;
+        let entry3 = Arc::new(AccountStorageEntry::new(
+            dummy_path, dummy_slot, dummy_id3, dummy_size,
+        ));
+        entry3.alive_bytes.store(1000, Ordering::Relaxed);
```

Comment:
the magic numbers here like `1000` are a bit hard to reason about, should this be a function of `dummy_size` and the previous `alive_bytes` in the previous entries.

Reply:
sure